### PR TITLE
qemu.test: qemu_img_convert_with_target_is_zero

### PR DESCRIPTION
**Pull request by XueqiangWei**
convert image with parameter --target-is-zero -n

Signed-off-by: Xueqiang Wei <xuwei@redhat.com>
ID: 1942220, 1942222, 1942223

### CODE REVIEW

**Comment by XueqiangWei** [comment]
depends on avocado-framework/avocado-vt/pull/2971

**Comment by XueqiangWei** [comment]
1-Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.qemu_img_convert_with_target_is_zero.to_raw.target_is_zero.q35 Finshed 2021-03-22 12:54:42 2021-03-22 12:58:37 PASS 235.344181
2-Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.qemu_img_convert_with_target_is_zero.to_qcow2.target_is_zero.q35 Finshed 2021-03-22 12:58:38 2021-03-22 13:01:55 PASS 197.429287

(1/1) Host_RHEL.m8.u4.product_av.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.qemu_img_convert_with_target_is_zero.to_raw.q35: PASS (120.00 s)
(1/1) Host_RHEL.m8.u4.product_av.luks.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.qemu_img_convert_with_target_is_zero.to_qcow2.q35: PASS (118.75 s)

**Comment by XueqiangWei** [comment]
@zhencliu Please help review it. Many thanks

File: qemu/tests/qemu_img_convert_with_target_is_zero.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    fail_on((process.CmdError,))(source.convert)(
+            source_params, root_dir, cache_mode,
+            source_cache_mode, skip_target_creation)
+
+    vm = img_utils.boot_vm_with_images(test, params, env, (convert_target,))
+    session = vm.wait_for_login()
+    logging.info("Verify md5 value of the temporary file")
+    img_utils.check_md5sum(guest_temp_file, md5sum_bin, session,
+                           md5_value_to_check=md5_value)
+    session.close()
```
Comment (zhencliu) [inline qemu/tests/qemu_img_convert_with_target_is_zero.py:60]: remove the converted image after testing?

Comment (XueqiangWei) [reply qemu/tests/qemu_img_convert_with_target_is_zero.py:60]: Yes, agree with you. I will update it. Thanks.

**Comment by XueqiangWei** [comment]
(1/1) Host_RHEL.m8.u4.product_av.qcow2.virtio_scsi.up.virtio_net.Guest.RHEL.8.4.0.x86_64.io-github-autotest-qemu.qemu_img_convert_with_target_is_zero.to_qcow2.q35: PASS (213.14 s)

@zhencliu Updated, please help review it again. Many thanks.

**Comment by zhencliu** [comment]
LGTM

**Pull request merged**